github/commitstatus.go

```diff
@@ -178,6 +182,12 @@ func httpRequestDo(req *http.Request) (httpResponse, error) {
 	response.rateLimitRemaining, _ = strconv.Atoi(resp.Header.Get("x-ratelimit-remaining"))
 	rateLimitReset, _ := strconv.Atoi(resp.Header.Get("x-ratelimit-reset"))
 	response.rateLimitReset = time.Unix(int64(rateLimitReset), 0)
+
+	// time.RFC1123 is the format of the Github date header
```

Comment:
time.RFC1123 is the format of the HTTP `Date` header

Reply:
Changed as suggested.